src/vaadin-pdf-viewer.html

```diff
@@ -95,6 +168,12 @@
           this.$.viewerContainer.addEventListener('pagesinit', () => {
             this._viewer.currentScaleValue = 'auto';
           });
+          this.$.viewerContainer.addEventListener('pagechange', (event) => {
+            this._currentPage = event.pageNumber;
+            this._updatePageNumberStates();
+          });
+          pdfjsLib.GlobalWorkerOptions.workerSrc =
+            '../../pdfjs-dist/build/pdf.worker.min.js';
           this.open(this.src);
```

Comment:
`src` is public API but it's only read once (on ready()). Shouldn't you be able to dynamically set the component `src` at any time? You'll need a property observer for that